scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -731,12 +731,30 @@ object FormatWriter {
     }
   }
 
-  private val trailingSpace = Pattern.compile("\\h+$", Pattern.MULTILINE)
+  /**
+    * No Unicode is necessary in the three (3) Pattern.compile() below.
+    * scalafmt parser will have already have objected.
+    * https://github.com/scala/scala/blob/2.11.x/spec/01-lexical-syntax.md
+    * gives the recognized scala whitespace characters.
+    * "Whitespace characters. \U0020 | \U0009 | \U000D | \U000A"
+    * (original text has lowercase u. U substituted here to allow scalafmt
+    *  to format this file.)
+    *
+    * Of the characters which the parser will pass, only space (\u0020)
+    * and tab (\u0009) are horizontal whitespace.
+    * Save execution time by not checking what the character could not be.
+    *
+    * */
```

Comment:
should this be `*/` or `**/` instead of `* */`?

Reply:
Being a visitor and it not being my usual coding style, for consistency I did a 
"monkey see, monkey do" of several other comments in the same file.

Yes, the trailing "* */" is not a variant I have seen in recent memory,

That comment is re-worked in the commit I am about to submit, using my
usual "//" style.  If the concept of the comment is good, I can adapt the 
comment style to anything you suggest.  Small dragons.